rust/namespace/src/utils.rs

```diff
@@ -14,8 +14,8 @@ pub fn assert_part_of_namespace<'a>(
     namespace: &Account<'a, Namespace>,
 ) -> Result<()> {
     let data = artifact.data.borrow_mut();
-    let number = u32::from_le_bytes(*array_ref![data, 8, 4]) as usize;
-    let offset = 12 as usize;
+    let number = u32::from_le_bytes(*array_ref![data, 9, 4]) as usize;
```

Comment:
Maybe a better fix would be to just move this function to the staking crate as the namespace crate doesn't use it. Thoughts?

Reply:
Also, I'm not a huge fan of pulling apart the data in this way, would be better to assert to a particular account and then grab the namespaces there, this is how I did it in namespaces: https://github.com/raindrops-protocol/raindrops/blob/master/rust/namespace/src/lib.rs#L257-L271

Reply:
Well, let's fix this bug and merge it first, we can improve it later.
What do you think?

Reply:
Works for me